src/applications/mhv-medical-records/tests/containers/LandingPage.unit.spec.jsx

```diff
@@ -187,8 +177,9 @@ describe('Landing Page', () => {
       }),
     ).to.exist;
     expect(
-      screen.getByRole('link', {
-        name: 'Go to manage your electronic sharing settings',
+      screen.getByText('Share personal health data with your care team', {
+        selector: 'h2',
+        exact: true,
```

Comment:
Aren't we still showing "Go to manage your electronic sharing settings" on the landing page? Can we just keep this test and add yours instead of replacing it?

Reply:
fixed it. Removed by mistake.